tasks/sync-granule/index.js

```diff
@@ -127,7 +134,23 @@ function syncGranule(event) {
     const granuleDuplicates = {};
     const granules = [];
     granuleResults.forEach((gr) => {
-      granules.push(gr.ingestedGranule);
+      if (!gr.ingestedGranule.createdAt) {
+        const createdAt = new Date().getTime();
+        gr.ingestedGranule.createdAt = createdAt;
+        granules.push(gr.ingestedGranule);
+        // update granule to ensure createdAt date ends up in DB
+        updateGranule({
+          prefix: event.config.stackName,
+          body: {
+            collectionId: constructCollectionId(
+              gr.ingestedGranule.dataType,
+              gr.ingestedGranule.version
+            ),
+            granuleId: gr.ingestedGranule.granuleId,
+            createdAt: gr.ingestedGranule.createdAt,
+          },
+        });
+      }
```

Comment:
I thought,
1. createdAt comes from the step function workflow_start_time (it's available from cumulus message)
2. add the `createdAt` to each granule in the output payload
Maybe different decision was made?

Reply:
oh no. 🤦🏻‍♂️ I probably misunderstood. 

I thought the workflow_start_time was used by `write_granules` and my using "now()" was intended to give a "close enough" createdAt date/time. I included a call to `updateGranule` to bring consistency with `queueGranules` task.

Reply:
Can we talk about this in the architecture meeting? I feel like I need a better understanding of what we're trying to do here, because I'm not sure if we want to be doing an `updateGranule` call in this lambda or not

Reply:
The call to `updateGranule` will be removed. My understanding was very poor. I spoke with Jenny this morning and I am going to simply use the `workflow_start_time` rather than using `now()`.